graylog2-web-interface/src/components/configurations/PluginsConfig.tsx

```diff
@@ -60,6 +60,7 @@ const PluginsConfig = () => {
 
   useEffect(() => {
     const pluginPromises = pluginSystemConfigs
+      .filter((systemConfig) => !systemConfig?.isNotClusterConfig)
```

Comment:
We have a negation (`Not`) in the naming and negate it again here. I would suggest we name the flag `isClusterConfig` instead.

Reply:
In this case, I would have to add the `isClusterConfig` flag to every place where we use SystemConfiguration. Or condition should be `systemConfig?.isClusterConfig === false` or I can change the flag name to `skipClusterConfigRequest`. What would you prefer?

Reply:
From the given options, I would prefer `skipClusterConfigRequest`.